client/components/NewErrand/NewErrandContainer.jsx

```diff
@@ -66,21 +66,19 @@ const NewErrandContainer = ({ navigation }) => {
 
           <View style={styles.buttons}>
             <View style={styles.cancel}>
-              <Button title="Cancel" />
+              <Button
+                title="Cancel"
+                onPress={() => {
+                  navigation.navigate('Dashboard');
+                }}
+              />
             </View>
             <View style={styles.submit}>
               <Button
                 title="Submit"
                 onPress={() => {
                   setNewErrand(newErrandObj);
```

Comment:
instead we should append to the existing _all_ errands recoil state
```js
import errandState from '../../state/atoms/errands';
...
const setErrands = useSetRecoilState(errandState);
...

<Button
  title="Submit"
  onPress={() => {
    setErrands((oldErrands) => [
      ...oldErrands,
      newErrandObj,
    ]);
    addToMongo(newErrandObj); 

```

Reply:
i got this directly from the recoil documentation: https://recoiljs.org/docs/basic-tutorial/atoms (see TodoItemCreator example)